### ISSUE TITLE
If you cancel an upgrade, LT thinks it's been upgraded anyway

### ISSUE THREAD

**Issue by rmunn** [LightTable/LightTable#1735]
If you click the "cancel" button when LightTable offers to upgrade to a new version, you won't be able to upgrade until you quit and restart LightTable. Here are two different scenarios:

1) Start LightTable 0.6.7. Wait for it to check for updates and say "There's a newer version of LightTable!", and click the Cancel button instead of the "Download and install" button. Now click on the `changelog` link, and it will say that your current version of LT is 0.7.1 -- and clicking "Check for updates" at the bottom of the changelog window will say "At latest version: 0.7.1". Quitting and restarting LightTable will "remind" it that it's still just 0.6.7 and hasn't been upgraded yet.

2) Start LightTable 0.6.7, and quickly, before it's had time to check for updates, click on the `changelog` link. The current version of LT will be correctly listed as 0.6.7, and you'll be able to click "Check for updates" and get offered the new update. Cancel it, and click the button again. The version number at the top of the window will still say 0.6.7, but the message at the bottom will say "At latest version: 0.7.1". Closing and re-opening the changelog window will make the version at the top of the window say "0.7.1", incorrectly. Only quitting and re-opening LightTable will it remember that the version number is still 0.6.7.

In short, checking for updates is incorrectly setting LT's current version number.
